src/app-layout/visual-refresh-toolbar/drawer/global-drawer.tsx

```diff
@@ -69,6 +73,11 @@ function AppLayoutGlobalDrawerImplementation({
   const lastOpenedDrawerId = drawersOpenQueue.length ? drawersOpenQueue[0] : null;
   const hasTriggerButton = !!activeGlobalDrawer?.trigger;
 
+  useEffect(() => {
+    onMount?.();
```

Comment:
It looks like a literal implementation of this anti-pattern: https://react.dev/learn/you-might-not-need-an-effect#notifying-parent-components-about-state-changes

Reply:
addressed